plugins/registry.js

```diff
@@ -155,7 +161,7 @@ export default class PluginRegistry {
     // - text - A string or React element to display in the menu
     // - action - A function to trigger when component is clicked on
     // Returns a unique identifier.
-    registerPostDropdownMenuAction(text, action) {
+    registerPostDropdownMenuAction(text, action, filter) {
```

Comment:
Can you add a comment describing what filter does?

Reply:
Ooopss, I missed this one.  I'll update.  Thanks